larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -135,6 +142,20 @@ void* Init(TfLiteContext* context, const char* buffer, size_t length) {
     conv_params->padding_type = kTfLitePaddingSame;
   } else {
     context->ReportError(context, "Invalid padding attribute.");
+  }
+
+  // Read fused activation
+  if (m["activation"].IsNull() || m["activation"].ToString() == "" ||
+      m["activation"].ToString() == "NONE") {
+    conv_params->activation = kTfLiteActNone;
+  } else if (m["activation"].ToString() == "RELU") {
+    conv_params->activation = kTfLiteActRelu;
+  } else if (m["activation"].ToString() == "RELU1") {
```

Comment:
Are you sure that `RELU1` is already supported in TF 2.1? I think support in the converter was only added in TF 2.2.

Reply:
Since all relu's are implemented by clamping, they are trivially supported by the kernel code. They were indeed not yet supported by the converter.

Reply:
Makes sense :+1: